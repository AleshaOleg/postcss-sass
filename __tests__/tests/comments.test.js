var getPostCssTreeFromSass = require('../../helpers/getPostCssTreeFromSassTree');

it('comments.sass', function () {
    expect(getPostCssTreeFromSass('comments')).toMatchSnapshot();
});
