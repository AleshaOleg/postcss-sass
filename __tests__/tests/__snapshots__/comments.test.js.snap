exports[`test comments.sass 1`] = `
Object {
  "nodes": Array [
    Object {
      "nodes": Array [
        Object {
          "raws": Object {
            "before": "
    ",
            "commentType": "single",
            "left": " ",
            "right": "",
          },
          "text": "Comment1",
          "type": "comment",
        },
        Object {
          "prop": "color",
          "raws": Object {
            "before": "
    ",
            "between": ": ",
            "semicolon": false,
          },
          "source": Object {
            "end": Object {
              "column": 14,
              "line": 3,
            },
            "input": Input {
              "css": ".comments
    // Comment1
    color: red
    /* Comment2 */
    background-color: blue
",
              "id": "<input css 1>",
            },
            "start": Object {
              "column": 5,
              "line": 3,
            },
          },
          "type": "decl",
          "value": "red",
        },
        Object {
          "raws": Object {
            "before": "
    ",
            "commentType": "multi",
            "left": " ",
            "right": " ",
          },
          "text": "Comment2",
          "type": "comment",
        },
        Object {
          "prop": "background-color",
          "raws": Object {
            "before": "
    ",
            "between": ": ",
            "semicolon": false,
          },
          "source": Object {
            "end": Object {
              "column": 26,
              "line": 5,
            },
            "input": Input {
              "css": ".comments
    // Comment1
    color: red
    /* Comment2 */
    background-color: blue
",
              "id": "<input css 1>",
            },
            "start": Object {
              "column": 5,
              "line": 5,
            },
          },
          "type": "decl",
          "value": "blue",
        },
      ],
      "raws": Object {
        "before": "",
        "between": "",
      },
      "selector": ".comments",
      "source": Object {
        "end": Object {
          "column": 27,
          "line": 5,
        },
        "input": Input {
          "css": ".comments
    // Comment1
    color: red
    /* Comment2 */
    background-color: blue
",
          "id": "<input css 1>",
        },
        "start": Object {
          "column": 1,
          "line": 1,
        },
      },
      "type": "rule",
    },
  ],
  "raws": Object {
    "before": "",
    "semicolon": undefined,
  },
  "source": Object {
    "end": Object {
      "column": 27,
      "line": 5,
    },
    "input": Input {
      "css": ".comments
    // Comment1
    color: red
    /* Comment2 */
    background-color: blue
",
      "id": "<input css 1>",
    },
    "start": Object {
      "column": 1,
      "line": 1,
    },
  },
  "type": "root",
}
`;
